ibmvmc/ibmvmc.c

```diff
@@ -1743,6 +1808,10 @@ static int ibmvmc_probe(struct vio_dev *vdev, const struct vio_device_id *id)
 	dev_set_drvdata(&vdev->dev, adapter);
 
 	return 0;
+
+crq_failed:
+	kthread_stop(adapter->reset_task);
+	return -1;
```

Comment:
What's the reasoning behind this? If we reach this point, reset_task was started correctly and wasn't the cause of failure.